<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-9O8PPK85" name="com.div.proj.testcases.ShopAddToCartTest" tests="1" failures="1" timestamp="2021-08-15T02:27:54 AEST" time="31.668" errors="0">
  <testcase name="ShopAddToCartTest" time="31.668" classname="com.div.proj.testcases.ShopAddToCartTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Funny Cow Quantity mismatch expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Funny Cow Quantity mismatch expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.div.proj.testcases.ShopAddToCartTest.ShopAddToCartTest(ShopAddToCartTest.java:58)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:77)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:15)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </failure>
  </testcase> <!-- ShopAddToCartTest -->
  <system-out/>
</testsuite> <!-- com.div.proj.testcases.ShopAddToCartTest -->
